js/ui/pivot_grid/ui.pivot_grid.area_item.js

```diff
@@ -401,29 +402,25 @@ export const AreaItem = Class.inherit({
     },
 
     groupWidth: function(value) {
-        if(value === undefined) {
-            return this._groupElement.width();
-        } else if(value >= 0) {
-            this._groupWidth = value;
-            return (this._groupElement[0].style.width = value + 'px');
-        } else {
-            return (this._groupElement[0].style.width = value);
+        if(!isDefined(value)) {
+            return;
         }
+
+        this._groupWidth = isNumeric(value) ? value : null;
+        this._setGroupElementDimension('width', value);
     },
 
     groupHeight: function(value) {
-        if(value === undefined) {
-            return this._groupElement.height();
+        if(!isDefined(value)) {
```

Comment:
It looks like it was used to READ height value: "item.height()" will return it.
Now, it returns 'undefined' in the same scenario and 'groupHeight' looks like 'setGroupHeight' only.

Reply:
Fixed in https://github.com/DevExpress/DevExtreme/pull/18458/commits/492a8de747bc7aa5f84c9cdcd28425a8e6f9fd5c